sos/cleaner/mappings/hostname_map.py

```diff
@@ -50,10 +50,14 @@ def load_domains_from_map(self):
         in this parser, we need to re-inject entries from the map_file into
         these dicts and not just the underlying 'dataset' dict
         """
-        for domain in self.dataset:
+        for domain, ob_pair in self.dataset.items():
             if len(domain.split('.')) == 1:
                 self.hosts[domain.split('.')[0]] = self.dataset[domain]
             else:
+                if ob_pair.startswith('obfuscateddomain'):
```

Comment:
Not required now, but in future, it would make sense to use a constant for `obfuscateddomain`, to declare it just once and use any times in code (this is 2nd usage).